src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       href="#logo"
+       class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Here, `logo_image` seems to be more suitable for a img tag. Consider changing it to `logo__image` or similar.